aws/manifest.yml.tpl

```diff
@@ -55,10 +57,8 @@ jobs:
 
   networks:
   - name: private
-    static_ips: [10.0.0.6]
+    static_ips: [${aws_static_ip}]
     default: [dns, gateway]
-  - name: public
-    static_ips: [${aws_static_ip}] # <--- Replace with Elastic IP
```

Comment:
This static IP is required by the microbosh so that it can communicate with AWS in order to spin up new VMs, upload stemcells etc.